.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-  "prettier.configPath": ".prettierrc.json"
+  "prettier.configPath": "./apps/site/.prettierrc.json"
```

Comment:
Since this config is now being prefixed with `apps/site/`, shouldn't it live in the workspace-specific settings (`apps/site/.vscode/settings.json`) rather than the root?